src/main/webapp/data-visualization.js

```diff
@@ -72,8 +72,10 @@ function displayAmChartsMap(data, locationInfo, description, geoData, color) {
     chart.goHome();
   });
 
+  const isCountyQuery = data[0].id.indexOf('US') === -1;
+
   // Albers projection for state level, Mercator for county level
-  if (data[0].id.indexOf('US') !== -1) {
+  if (!isCountyQuery) {
```

Comment:
Nit: any reason not to swap these and avoid negating the boolean?

Reply:
Fixed!